packages/server-boilerplate-middleware/src/index.ts

```diff
@@ -94,6 +100,23 @@ export const createServer = ({
       }),
     );
   }
+  if (enableXray) {
+    logger.silly('enable aws xray tracing');
+    
+    // Gives you more information about the node and container
+    AWSXRay.config(xrayOptions.config || DEFAULT_XRAY_CONFIG);
+    
+    AWSXRay.setLogger(logger);
+    AWSXRay.setDaemonAddress(xrayOptions.daemonAddress || DEFAULT_XRAY_DAEMON_ADDRESS);
+    
+    // Capture all outgoing https and http requests
+    AWSXRay.captureHTTPsGlobal(require('https'));
+    AWSXRay.captureHTTPsGlobal(require('http'));
+
+    // You can override the default service name that you define in code with the AWS_XRAY_TRACING_NAME environment variable.
+    app.use(AWSXRay.express.openSegment(DEFAULT_AWS_XRAY_TRACING_NAME));
```

Comment:
how about having it in `xrayOptions` as well? So that all config is done in 1 place

Reply:
updated